packages/atlas/src/views/studio/MyVideosView/MyVideosView.tsx

```diff
@@ -80,6 +80,7 @@ export const MyVideosView = () => {
       : _drafts.slice().sort((a, b) => new Date(a.updatedAt).getTime() - new Date(b.updatedAt).getTime())),
   ]
 
+  useChannelNfts(activeChannelId)
```

Comment:
Why is this here?

Reply:
The NFTs should be fetched as part of the query that fetches videos already, no?

Reply:
Yeah but we need to fetch it one more time in `useGetNftSlot`